JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -84,13 +81,14 @@ import kotlinx.coroutines.runBlocking
  * @param postsRepository data source for this screen
  * @param scaffoldState (state) state for the [Scaffold] component on this screen
  */
+@OptIn(ExperimentalMaterialApi::class)
 @Composable
 fun HomeScreen(
     navigateTo: (Screen) -> Unit,
     postsRepository: PostsRepository,
     scaffoldState: ScaffoldState = rememberScaffoldState()
 ) {
-    val (postUiState, refreshPost, clearError) = launchUiStateProducer(postsRepository) {
+    val (postUiState, refreshPost, clearError: () -> Unit) = launchUiStateProducer(postsRepository) {
```

Comment:
Was this explicit type added intentionally? Seems unbalanced only having it on one property.

Reply:
Left it accidentally. It's too much magic